prow/plugins/trigger/trigger.go

```diff
@@ -115,3 +115,19 @@ func trustedOrgForRepo(config *plugins.Configuration, org, repo string) string {
 	}
 	return org
 }
+
+func isUserTrusted(ghc githubClient, user, trustedOrg, org string) (bool, error) {
+	orgMember, err := ghc.IsMember(trustedOrg, user)
+	if err != nil {
+		return false, err
```

Comment:
No matter what err is here, if it errors out earlier, we will miss a chance to check if the user is a member of `org` (they may be).

Reply:
This is no different from what happened before so I am fine with it.

Reply:
:)